content/en/blog/news/2022-gsoc-cbp.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "GSoC’22 Community Bonding Period @Jenkins X"
```

Comment:
Instead of `at` and `@`, `-` more sense to me, but `@` is what I saw on most of the blogs :innocent:
```suggestion
title: "GSoC’22 Community Bonding Period - Jenkins X"
```

Reply:
I am fine with `-`. Also why the extra `'` between GSoC and 22?

Reply:
I think it's trending, no specific reason, 
```
GSoC 2022 Community Bonding Period with Jenkins X
```
this also looks fine to me, but it's a bit plain title.

Reply:
I am a simple man, boring things look awesome to me :grimacing:

Reply:
done